include/sys/emmc.h

```diff
@@ -0,0 +1,279 @@
+#ifndef _SYS_EMMC_H_
```

Comment:
Should be `_DEV_EMMC_H_`.

Reply:
Ideally yes. We should move some of the headers from `sys` to `dev`. However there's a small detail that makes it a bit awkward - I think that stuff like `sys/emmc.c` and `sys/isa.c` should be in a separate catalogue as well, but we don't have a `dev` catalogue for `.c` files, only for headers.

I'm keeping it this way until this gets cleared up.